examples/examples.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright (c) 2017 VMware, Inc. All Rights Reserved.
-
```

Comment:
Can you revert these whitespace changes here and the other 2 files below?

Reply:
I reset to the previous commit e just sent now the perfmanager example.